common/src/WooCommerce.php

```diff
@@ -148,6 +153,22 @@ private function sendInvite($order_id) {
 
         // send invite
         $api = new API($api_domain, $shop_id, $api_key);
+        try {
+            if ($this->plugin->getOption('invite') == self::POPUP_OPTION && !$api->hasConsent($data)) {
+                $this->insert_comment($order_id, 'Invite was not send as customer did not consent.');
+                return;
+            }
+        } catch (WebwinkelKeurAPIError $e) {
+            $this->logApiError($e);
+            $this->insert_comment(
+                $order_id,
+                sprintf(
+                    __('The %s invitation could not be sent.', 'webwinkelkeur'),
+                    $this->plugin->getName()
+                ) . ' ' . $e->getMessage()
```

Comment:
Instead of `sprintf` and then `.  '  '  .`, just put everything inside the `sprintf`